### PR TITLE
Email Forwards: Bump the number of email forwards allowed for business users.

### PR DESCRIPTION
**Pull request by matthusby**

### CODE REVIEW

**Comment by bikedorkjon** [comment]
Looks good works as expected :+1:

File: client/my-sites/upgrades/domain-management/email-forwarding/email-forwarding-limit.jsx [hunk 1]
```diff
@@ -4,6 +4,14 @@
 import React from 'react';
 
 const EmailForwardingLimit = React.createClass( {
+	maxForwards() {
+		if ( this.props.maxForwards === 500 ) {
+			return 'unlimited';
```
Comment (gziolo) [inline client/my-sites/upgrades/domain-management/email-forwarding/email-forwarding-limit.jsx]: Shouldn't we translate it?

**Comment by mtias** [comment]
@matthusby what is the status of this one?

**Comment by matthusby** [comment]
I just made one small update.  My calypso is totally borked right now and I don't have time to find out whats going on right now.  Could someone take this over? @scruffian @drewblaisdell  Should just need testing.

**Comment by jeremeylduvall** [comment]
Just noting that I opened #3156 to serve the same purpose (didn't know this existed). Definitely only need one or the other.

**Comment by matthusby** [comment]
Thanks @jeremeylduvall I will close this one.

**Pull request closed without merging**